### ISSUE TITLE
Bug Report: Duplicate Labels in Command Palette

### ISSUE THREAD

**Issue by francisco-milan** [abapGit/abapGit#6822]
Dear abapGit Team,

I wanted to bring to your attention a bug that I've encountered in the Command Palette. When assigning the same label to multiple repositories, I've noticed that the Command Palette displays duplicate entries for those labels. In my case, I've observed duplicate entries for labels such as "FM", "abapGit", and "abap2xlsx".

![image](https://github.com/abapGit/abapGit/assets/36721657/e8a47660-5850-42f2-8cc2-cc69bdc7eda9)

After investigating the issue, I've identified a solution that can prevent these duplicates from appearing in the Command Palette. By implementing a check to ensure that only items with unique titles are included in the items array, we can effectively resolve this issue.

To implement this fix, I've added the following lines of code:

```
// Check if the title already exists before adding to items array
var titlesAdded = {};
items = items.filter(function(item) {
  if (!titlesAdded[item.title]) {
    titlesAdded[item.title] = true;
    return true;
  }
  return false;
});
```
This is the result after applying the fix.

![image](https://github.com/abapGit/abapGit/assets/36721657/3c6594fc-e3ee-4963-a1c8-48a38a6bb2fb)

To expedite the resolution of this issue, I've already created a pull request with the proposed fix (#6821)

Please let me know if you need any further information or assistance with addressing this bug. Thank you for your attention to this matter.

Best regards,

Francisco Milan.

**Comment by sbcgua** [comment]
Sorry for some critique. But for me, it again looks like a patch instead of semantics - https://github.com/abapGit/abapGit/pull/6808#issuecomment-1937794150

The right solution is to mark labels in table raws as unlisted. Instead of deduplication, which does not understand the context.
Just imho.

**Comment by mbtools** [comment]
Closed by #6821
